server/services/datacommons.py

```diff
@@ -88,11 +88,12 @@ def search(query_text, max_results):
     return response.json()
 
 
-def get_stats(place_dcids, stats_var):
+def get_stats(place_dcids, stats_var, obs_dates='latest'):
```

Comment:
please undo this change (obs_date is not an accepted parameter for the rest api)